openwisp_notifications/models.py

```diff
@@ -75,7 +87,7 @@ def send_email_notification(sender, instance, created, **kwargs):
     ):
         return
     # send email
-    subject = instance.data.get('email_subject', instance.description[0:24])
+    subject = instance.data.get('email_subject', instance.short_description[0:24])
```

Comment:
I don't understand this. Why don't we dedicate a part of the notification configuration to the subject?

Reply:
It was implemented like this in `openwisp_monitoring.notifications`. I will add a configurable setting for subject as well. 
**Questions**
- Do we have to truncate length of subject or just inform user in the documentation as best practices for giving subject settings? 
- Do we have make it accept variables as it is done in notification message ?

Reply:
`django-notifications` provides a `JSONField` - `data`, email's subject is stored in that field.  Would you like to have a dedicated field for the subject?

Reply:
> Would you like to have a dedicated field for the subject?

Can we get the subject from the type configuration instead? So we don't avoid writing the same content in the DB zillions of time.
The subject will be always the same, the only part that changes are variables and can be passed in.